cli/cmd/vulnerability.go

```diff
@@ -371,6 +399,34 @@ func pollScanStatus(requestID string) error {
 	}
 }
 
+func vulnFailureThreshold(assessment api.HostVulnHostAssessment) {
+	severityRating, _ := eventSeverityToProperTypes(assessment.HighestSeverity())
```

Comment:
Move to vulnerabilities_host.go as` func (assessment *HostVulnHostAssessment) vulnFailureThreshold()`